packages/react/src/ActionMenu/ActionMenu.examples.stories.tsx

```diff
@@ -656,3 +656,102 @@ export const DynamicAnchorSides = () => {
     </>
   )
 }
+
+export const AnchorPositionScrollBug = () => {
+  return (
+    <>
+      {/* First Box: 70% of page height */}
+      <Box
+        sx={{
+          height: '70vh',
+          backgroundColor: 'canvas.subtle',
+          p: 3,
+          border: '1px solid',
+          borderColor: 'border.default',
+        }}
```

Comment:
sx is deprecated, can we use a `div` with styles or a css modules class instead?